### ISSUE TITLE
AndroidX migration?

### ISSUE THREAD

**Issue by nareshkatta99** [prolificinteractive/material-calendarview#911]
The users who are Using AndroidX can not use this awesome library.
Please create another version for AndroidX migrated  projects.

**Comment by mpierucci** [comment]
Agree, Im very tempted to use this library, but Ill have to migrate to Rx soon and thats an issue.

**Comment by aldebaran798** [comment]
Yes please! This would be great!

**Comment by artnest** [comment]
It works with AndroidX. Right now I'm using AS 3.3 with AGP 3.3.0 and Jetifier 1.0.0-beta02 (it was included for a fix with another library, don't remember exactly).

**Comment by barnhill** [comment]
@nareshkatta99 you probably just need to set these in your gradle.properties

android.useAndroidX=true
android.enableJetifier=true

We are using this library and are using android X with Jetifier.  This request is invalid but moving to newer AndroidX libraries would still be a good idea for future support library enhancements and updates.

**Comment by nareshkatta99** [comment]
In some mobiles, I think, there is  an issue while using this library in fragments.
When using the library in Fragment, it says that android.support.v4.app.Fragment can not be found.
The reason is that if you use androidx you are using androidx.fragment.app.Fragment the corresponding fragment.

**Comment by barnhill** [comment]
Are you using jetifier?
